packages/core/src/components.d.ts

```diff
@@ -12,738 +12,738 @@ export { IconProps } from "./icons";
 export { Color, Mode, TextFieldTypes } from "@ionic/core";
```

Comment:
Just lint modifications.

Reply:
This file is generated automatically, this lint modification will be lost